README.md

```diff
@@ -27,17 +25,35 @@ To run the site locally:
    ```
```

Comment:
Not something you did, but I think we can get rid of step 2 here because it's captured in current step 3.

Reply:
Thanks! Added your suggestions in